example/index.html

```diff
@@ -6,7 +6,7 @@
     <script src="js/aframe.min.js"></script>
   </head>
   <body>
-    <a-scene stats>
+    <a-scene stats debug="true">
```

Comment:
I've opened and issue so we could discuss it there https://github.com/aframevr/aframe-inspector/issues/367